### DISCUSSION TITLE
Bifacial Reference Modules

### DISCUSSION THREAD

**Discussion by kurt-rhee** [pvlib/pvlib-python#1858] [Q&A]
Hello PVLIB community, 

I was wondering, are there any references out there for models which might decompose the total irradiance hitting a bifacial module into the frontside and backside components?  Something like Module Current > Total Irradiance > Frontside Irradiance & Backside Irradiance.

**Comment by jules-ch** [comment]
pvlib comes with bifacial modeling for infinite sheds : https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.bifacial.infinite_sheds.get_irradiance.html

This might be useful for your use case.

**Comment by kandersolar** [comment]
I'm not aware of such a model.  Essentially this would be a bifacial GTI-DIRINT, coupled with a step to estimate irradiance from current. 

Perhaps you could cobble together a numerical approach using "forward" models as described in https://github.com/pvlib/pvlib-python/discussions/1305.

**Comment by cwhanse** [comment]
> Total Irradiance > Frontside Irradiance & Backside Irradiance.

I think this step will be difficult without additional data, as there are many combinations of front and rear irradiance that add to the same total, resulting from different reflection and shadow geometries. Maybe a simulation of clear-sky front and rear irradiance could help set masks for when shading is more or less important, and these masks could condition the separation of total into front/rear.

**Comment by kurt-rhee** [comment]
Thank you so much @jules-ch @kandersolar @cwhanse for the notes and feedback.
